flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/CodeAnalyzer.java

```diff
@@ -384,6 +387,22 @@ private void checkForNewConnection(NewExpressionNode newExpressionNode,
         }
     }
 
+    @Override
+    public void visit(BracedExpressionNode bracedExpressionNode) {
+        bracedExpressionNode.expression().accept(this);
+    }
```

Comment:
I think this is implicitly done by the visitor pattern.

Reply:
removed